app/views/layouts/application.html.erb

```diff
@@ -43,9 +43,11 @@
         </div>
         <div class="govuk-header__content">
           <a href="/" class="govuk-header__link govuk-header__link--service-name">
-            Apply for postgraduate teacher training
+            <%= t('layout.service_name') %>
           </a>
-          <p><%= current_candidate.email_address unless current_candidate.nil? %></p>
+          <% if current_candidate.present? %>
```

Comment:
```suggestion
          <% if candidate_signed_in? %>
```

Thinking we could the Devise's helper for checking a candidate is signed in?

Reply:
✅